clients/python/raccoon_client/rest/client.py

```diff
@@ -0,0 +1,68 @@
+import time
+import uuid
+
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3 import Retry
+
+from raccoon_client.client import Client, Event
+from raccoon_client.protos.raystack.raccoon.v1beta1.raccoon_pb2 import SendEventRequest, SendEventResponse, Event as EventPb
+from raccoon_client.rest.option import RestClientConfig
+from raccoon_client.serde.serde import Serde
+from raccoon_client.serde.util import get_serde, CONTENT_TYPE_HEADER_KEY, get_wire_type
+from raccoon_client.serde.wire import Wire
+
+
+class RestClient(Client):
+    session: requests.Session
+    serde: Serde
+    wire: Wire
+
+    def __init__(self, config: RestClientConfig):
+        self.session = requests.session()
+        self.url = config.url
+        self.serde = get_serde(config.serialiser)
+        self.wire = get_wire_type(config.wire_type)
+        self.headers = self._set_content_type_header(config.headers)
+        self._set_retries(self.session, config.max_retries)
+        self.timeout = config.timeout
+
+    def _set_retries(self, session, max_retries):
+        retries = Retry(
+            total=max_retries,
+            backoff_factor=1,
+            status_forcelist=[500, 502, 503, 504, 521, 429],
+            allowed_methods=["POST"],
+            raise_on_status=False,
+        )
+        session.mount("https://", HTTPAdapter(max_retries=retries))
```

Comment:
Raccoon can be hosted on both http and https setup. 
Ex: Since aurora is https, the browser won't allow any requests to an http endpoint from the page, hence attaching both.